packages/manager/modules/nutanix/src/dashboard/general-info/template.html

```diff
@@ -0,0 +1 @@
+<div>coming soon...</div>
```

Comment:
No translations for that ?

Reply:
its just a layout text, the feature is not implemented, it will be replaced  by actual text.